src/Symfony/Component/DependencyInjection/Tests/Dumper/YamlDumperTest.php

```diff
@@ -56,4 +54,9 @@ public function testAddService()
             $this->assertEquals('Unable to dump a service container if a parameter is an object or a resource.', $e->getMessage(), '->dump() throws a RuntimeException if the container to be dumped has reference to objects or resources');
         }
     }
+
+    private function assertDumpedYamlEqualsExpectedStructure($yaml, $expected, $message = '')
```

Comment:
I like `assertEqualYamlStructure()`. Changed it everywhere to use this method name.